kedro_benchmarks/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[project]
+name = "kedro_benchmarks"
```

Comment:
The reason to do this is `multiprocessing` breaks imports. Without this I get a `benchmarks` module is not found error.

To make sure this always in `sys.path`, I add this into part of the installation step. Other suggestions are welcomed.

I have tried `PYTHONPATH` as well but it doesn't seem to work as new process doesn't inherit it.